### PR TITLE
[8.0] Logistics Order - UI Improvements

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: logistic_order/model/sale_order.py [hunk 1]
```diff
@@ -30,3 +30,37 @@ class SaleOrder(models.Model):
...
+        related='pricelist_id.currency_id',
+        co_model='res.currency',
+        string='Currency')
+    remark = fields.Text('Remarks / Description')
+
+    @api.multi
+    def action_quotation_send(self):
+        """ Add the action we want to perform in case the user
+        complete finally sends an email with designed wizard.
+
```
Comment (lepistone) [inline logistic_order/model/sale_order.py:51]: Not very clear to me, could you rephrase that? You mean that if the SO is marked as cost_estimate_only, it will be set do `done` immediately?

File: logistic_order/model/sale_order.py [hunk 2]
```diff
@@ -30,3 +30,37 @@ class SaleOrder(models.Model):
...
+class mail_compose_message(models.Model):
+    _inherit = 'mail.compose.message'
+
+    @api.multi
+    def send_mail(self):
+        context = self.env.context
+        if (context.get('default_model') == 'sale.order'
+                and 'default_res_id' in context
+                and 'mark_cost_estimate_as_done' in context):
+            res_id = context.get('default_res_id')
```
Comment (lepistone) [inline logistic_order/model/sale_order.py:72]: also here it's probably worth saying what is going on

**Comment by lepistone** [comment]
I'd add some doc on `cost_estimate_only`. I started with two notes on docstrings, but maybe it is worth saying what that does directly somewhere visible by the user, like the field tooltip.

Otherwise :+1: , thanks

**Comment by yvaucher** [comment]
@lepistone thanks doc improved
